src/nvim/eval.c

```diff
@@ -8684,7 +8684,7 @@ static void f_exists(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   const char *p = tv_get_string(&argvars[0]);
   if (*p == '$') {  // Environment variable.
     // First try "normal" environment variables (fast).
-    if (os_getenv(p + 1) != NULL) {
+    if (os_env_exists(p + 1)) {
```

Comment:
in `os_getenv` docstring , let's add:

    /// @see os_env_exists

Reply:
Will do that in https://github.com/neovim/neovim/pull/10655 - or as a followup to https://github.com/neovim/neovim/pull/2574#issuecomment-516342975.